Dockerfile

```diff
@@ -27,7 +30,9 @@ ENV RUSTUP_MAX_RETRIES=10
 ENV RUST_BACKTRACE="short"
 
 COPY . .
-RUN cargo build --release --features sandbox
+
+# Use bash variable substitution to convert environment name to lowercase
+RUN bash -c 'cargo build --release --features ${RUN_ENV@L}'
```

Comment:
Honestly though, I'd prefer using lowercase names for `RUN_ENV` and config files (`development.toml` / `sandbox.toml` / `production.toml`) instead of doing such shell sorcery. For the config files, I'm even okay with using a single `config.toml` instead, if that's preferable.

I'm aware this would break a lot of things including sandbox, but I'd prefer doing it now rather than after we move to production. 

I've made the necessary changes in https://github.com/juspay/hyperswitch/commit/5863a988bc43e766831b0724b076ae752e7f1b8a, let me know what you think.

Reply:
Okay, I'll take it up on a separate PR.